docs/cluster_manifest.md

```diff
@@ -0,0 +1,215 @@
+Postgres Cluster Manifest
+=========================
+
+Individual postgres clusters are described by the Kubernetes *cluster manifest*
+that has the structure defined by the `postgres CRD` (custom resource
+definition). The following section describes the structure of the manifest and
+the purpose of individual keys. You can take a look at the examples of the
+[minimal](https://github.com/zalando-incubator/postgres-operator/blob/master/manifests/minimal-postgres-manifest.yaml)
+and the
+[complete](https://github.com/zalando-incubator/postgres-operator/blob/master/manifests/complete-postgres-manifest.yaml)
+cluster manifests.
+
+When Kubernetes resources, such as memory, CPU or volumes, are configured,
+their amount is usually described as a string together with the units of
+measurements. Please, refer to the [Kubernetes
+documentation](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/)
+for the possible values of those.
+
+Manifest structure
+------------------
+
+A postgres manifest is a `YAML` document. On the top level both individual
+parameters and parameter groups can be defined  Parameter names are written
+in camelCase.
+
+### Cluster metadata
+
+Those parameters are grouped under the `metadata` top-level key.
+
+* **name**
+  the name of the cluster. Should start with the `teamId` followed by a dash.
+  Changing it after the cluster creation is not supported. Required field.
+
+* **namespace**
+  the namespace where the operator creates its Kubernetes objects (i.e. pods,
+  services, secrets) for the cluster. Changing it after the cluster creation
+  results in deploying or updating a completely separate cluster in the target
+  namespace. Optional (if present, should match the namespace where the
+  manifest is applied). 
+
+### Top-level parameters
+
+* **teamId**
+  name of the team the cluster belongs to. Changing it after the cluster
+  creation is not supported. Required field.
+
+* **dockerImage**
+  optional custom docker image that overrides the **docker_image** operator
```

Comment:
1. "optional" mentioned 2 times
2. Are there any specific  requirements on what should be in this image ? 
3. When present, this field overrides the operator k8s-cluster-wide setting for the image.